### ISSUE TITLE
Legend Missing from Fieldset for Number Per Page on Dashboard Works

### ISSUE THREAD

**Issue by julesies** [samvera/hyrax#2075]
Moved from https://github.com/curationexperts/nurax/issues/128
Originally reported by @mtribone 
**Descriptive summary**

The dropdown to set the number of results per page on the works dashboard has a fieldset, but no legend for the fieldset. If a higher description of the functionality of the controls is not necessary, then the fieldset should be removed.

Suggested text: In ScholarSphere we use <legend class="sr-only">Number of results to display per page</legend>

**Rationale**

A fieldset legend presents a description of the form elements within a fieldset and is especially useful to screen reader users. A legend should be provided when a higher level description is necessary for groups of check boxes, radio buttons, or other form controls.

**Expected behavior**

Expect all fieldsets to have legends for better accessibility.

**Actual behavior**

Fieldset doesn not have a legend.

**Steps to reproduce the behavior**

Navigate to your user dashboard
Select works from the left-hand side
Inspect the HTML associated with the number per page form controls
